restli-server/src/main/java/com/linkedin/restli/internal/server/util/MIMEParse.java

```diff
@@ -94,7 +108,14 @@ protected static ParseResults parseMimeType(String mimeType)
       String p = parts[i];
       String[] subParts = StringUtils.split(p, '=');
       if (subParts.length == 2)
+      {
         results.params.put(subParts[0].trim(), subParts[1].trim());
+      }
+      else
+      {
+        // invalid parameters
+        throw new InvalidMimeTypeException(mimeType);
+      }
```

Comment:
Is this needed? If existing behavior that ignores invalid params is not blocking you, then we can split the change and move the backwards incompatible change to a separate PR.